### PR TITLE
Look ma' no odm_georef

### PR DESCRIPTION
**Pull request by pierotofy**
This PR removes the `odm_georef` module altogether by switching to a geographic CRS early on in the processing pipeline (after the sparse reconstruction), thus avoiding the need for having two sets of textured models and the odm_georef module. This has obvious performance benefits and simplifies the processing pipeline quite a bit!

Still need to test multispectral datasets, split-merge and non-georeferenced datasets (and permutations of the above!)

### CODE REVIEW

**Comment by pierotofy** [comment]
Still to do: 
 - [x] Remove annoying debug messages from `opensfm compute_statistics`.

**Comment by pierotofy** [comment]
Preliminary tests all working.

**Comment by pierotofy** [comment]
I think this is ready to go! :boom:

File: stages/mvstex.py [hunk 1]
```diff
@@ -110,6 +110,15 @@ def add_run(nvm_file, primary=True, band=None):
                         '{nadirMode} '
                         '{labelingFile} '.format(**kwargs))
                 
+                # Backward compatibility: copy odm_textured_model_geo.mtl to odm_textured_model.mtl
+                # for certain older WebODM clients which expect a odm_textured_model.mtl
+                # to be present for visualization
+                # We should remove this at some point in the future
```
Comment (smathermather) [inline stages/mvstex.py:116]: Do you want me to open an issue for this just so we can stumble on it in 6-12 months? :D

**Comment by pierotofy** [comment]
Just kiddin'! forgot to test GCPs and found some errors. Now re-testing...

File: stages/mvstex.py (hunk 1 above)
Comment (pierotofy) [reply stages/mvstex.py:116]: :D sure! I'm probably going to forget otherwise.

**Comment by pierotofy** [comment]
Ok now it should be ready to go. :+1:

File: stages/mvstex.py (hunk 1 above)
Comment (smathermather) [reply stages/mvstex.py:116]: Lol. Sounds good.

Comment (smathermather) [reply stages/mvstex.py:116]: Yeah! I got issue 1234. That's auspicious: https://github.com/OpenDroneMap/ODM/issues/1234

**Comment by Saijin-Naib** [comment]
Dang, nearly 4k lines gone!

**Comment by pierotofy** [comment]
Branch passes the OATS test suite. I'll go ahead and merge. More manual testing to verify results is still appreciated.

**Pull request merged**